docs/CONFIG.md

```diff
@@ -3223,31 +3258,31 @@ LimitTransfer = 21_000 # Default
 ```
 LimitTransfer is the gas limit used for an ordinary ETH transfer.
 
-### BumpMin<a id='EVM-GasEstimator-BumpMin'></a>
+### BumpMin<a id='BumpMin'></a>
```

Comment:
Did you intend to change these?

Reply:
No! That was a mistake - an empty line in TOML that made it to belong to global scope. Thank you! Fixed.